src/seedsigner/views/seed_views.py

```diff
@@ -637,6 +645,7 @@ def run(self):
 
 
 class SeedBackupView(View):
+
```

Comment:
Added blank lines shouldn't be added to parts of the code that aren't being worked on for this PR.

Reply:
You'll probably find it easier to squash your branch's commits before doing a merge to catch up to the latest in `dev`. That way there's only one merge conflict review cycle.